app/controllers/application_controller.rb

```diff
@@ -2212,7 +2212,7 @@ def visibility_box_edit
 
   def get_record_display_name(record)
     return record.label                      if record.respond_to?("label")
-    return record.description                if record.respond_to?("description") && !record.description.nil?
+    return record.description                if record.respond_to?("description") && record.description.present?
```

Comment:
what happens if the description is very long?
also, can you please add a test :-) thanks!

Reply:
@ohadlevy We only check for presence or not here. TBH, flash_msg partial doesn't really check if a message fits in or not, never did. That said, not sure we ever checked for any data length for either name nor description field. I will play with the Add/Edit Custom Template form to see what happens, just curious how long a text string is "tolerated" by both, code and db engine. That said, its probably a different PR altogether if such an issue pops up.